gengokit/handler/handler_test.go

```diff
@@ -312,17 +314,17 @@ func TestPruneDecls(t *testing.T) {
 
 		// NewService returns a naïve, stateless implementation of Service.
 		func NewService() pb.ProtoServiceServer {
-			return generalService{}
+			return protoserviceService{}
```

Comment:
As a part of cleanup here, it would be good to replace:

`service ProtoService {` with `service Proto {` or something like that. 

As these tests look really weird with `protoserviceService`.